jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/bars/AppBarsTests.java

```diff
@@ -105,6 +119,14 @@ public void scrolledAppBarTests() {
         imageBar.is().notScrolled();
     }
 
+    @Test(description = "Test checks if element is short or not")
+    public void shortnessBarTests() {
+        shortBar.show();
+        shortBar.is().isShort();
+        outlinedBar.show();
+        outlinedBar.is().isUsual();
```

Comment:
Just to think about. If you use this check only here, maybe better name is 'notShort' or 'isUsualSize'?

Reply:
naming change

Reply:
done